### ISSUE TITLE
Test failure in newly ported CoreCLR test JIT._Methodical_tailcall_v4_hijacking_hijacking_._Methodical_tailcal

### ISSUE THREAD

**Issue by LLITCHEV** [dotnet/coreclr#2534]
Running CoreCLR tests after an update produces the following failure on Windows...
d:\repos\coreclr\tests\src\TestWrappers_x64_debug\JIT\JIT.XUnitWrapper.cs(174095): error : JIT._Methodical_tailcall_v4_hijacking_hijacking_._Methodical_tailcal
l_v4_hijacking_hijacking_cmd [FAIL] [D:\repos\coreclr\tests\runtest.proj]

  Raw output:      D:\repos\coreclr\bin\tests\Windows_NT.x64.debug\Reports\JIT\Methodical\tailcall_v4\hijacking\hijacking.output.txt
  To run the test:

> set CORE_ROOT=D:\repos\coreclr\bin\tests\Windows_NT.x64.debug\Tests\Core_Root
> D:\repos\coreclr\bin\tests\Windows_NT.x64.debug\JIT\Methodical\tailcall_v4\hijacking\hijacking.cmd

```
    Expected: True
    Actual:   False
      src\TestWrappers_x64_debug\JIT\JIT.XUnitWrapper.cs(174095,0): at JIT._Methodical_tailcall_v4_hijacking_hijacking_._Methodical_tailcall_v4_hijacking_h
```

  ijacking_cmd()

**Comment by hoyMS** [comment]
Not repro

**Comment by RussKeldorph** [comment]
Have repros under JitStress now.

JitStress=1
https://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/x64_checked_windows_nt_jitstress1/15/testReport/junit/JIT/_Performance_CodeQuality_Serialization_Deserialize_Deserialize_/_Performance_CodeQuality_Serialization_Deserialize_Deserialize_cmd/

JitStress=2
https://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/x64_checked_windows_nt_jitstress2/8/testReport/junit/JIT/_Methodical_tailcall_v4_hijacking_hijacking_/_Methodical_tailcall_v4_hijacking_hijacking_cmd/

**Comment by RussKeldorph** [comment]
@sivarv: hoping you can quickly assess whether this is a Test Issue.  If not, feel free to re-assign for full investigation.

**Comment by sivarv** [comment]
This test needs to be excluded from jitstress=1/2 runs as Jitstress could induce GS check that will disable tail calls.

**Comment by RussKeldorph** [comment]
Was fixed by #3659
